content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: IBM Cloud Private
+description: Example multicluster IBM Cloud Private install of Istio.
+weight: 70
+keywords: [Kubernetes,multicluster]
+---
+
+This example demonstrates how to use Istio's multicluster feature to join two
+[IBM Cloud Private](https://www.ibm.com/cloud/private) clusters together,
+using the [Kubernetes multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/).
+
+## Create the IBM Cloud Private Clusters
+
+1.  [Install two IBM Cloud Private servers](//www.ibm.com/support/knowledgecenter/en/SSBS6K_2.1.0.3/installing/installing.html).
+    __NOTE__: Make sure individual cluster Pod CIDR ranges and service CIDR ranges are unique and do not overlap
+    across the multicluster environment and may not overlap. This can be configured by `network_cidr` and
+    `service_cluster_ip_range` in `cluster/config.yaml`.
+    
+    {{< text bash >}}
+    ## Network in IPv4 CIDR format
+    network_cidr: 10.1.0.0/16
+
+    ## Kubernetes Settings
+    service_cluster_ip_range: 10.0.0.1/24
+    {{< /text >}}
+
+1.  After IBM Cloud Private cluster install finsihes, validate `kubectl` access to each cluster. In this example, consider
+    two clusters `cluster-1` and `cluster-2`.
+
+    [Configure cluster-1 with kubectl](//www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0.3/manage_cluster/cfc_cli.html).
+
+    1.  Check the cluster status:
+
+        {{< text bash >}}
+        $ kubectl get nodes
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+    1.  Repeat above steps to validate `cluster-2`.
+
+## Config Pod Communication Across IBM Cloud Private Clusters
+
+IBM Cloud Private uses Calico Node-to-Node Mesh by default to manage container networks. The BGP Client
+on each node distributes the IP router information to all nodes.
+
+To ensure pods can communicate across different clusters, you need to configure IP routers on all nodes
+in the cluster. You need two steps:
+
+1.  Add IP routers from `cluster-1` to `cluster-2`.
+
+1.  Add IP routers from `cluster-2` to `cluster-1`.
+
+You can check how to add IP routers from `cluster-1` to `cluster-2`. With Node-to-Node Mesh mode, each
```

Comment:
"You can check how to add IP routers from c1 to c2" doesn't make sense in this paragraph.  I'm not sure how to reword it in composition with the second sentence.  Perhaps switch the order of the sentences and change "You can check how" to " Validate ..."

Reply:
Done, updated to "You can check how to add IP routers from `cluster-1` to `cluster-2` to validate pod to pod communication across clusters."